paddle/cinn/README.md

```diff
@@ -0,0 +1,120 @@
+```
+                        ___                    ___          ___
+                       /\__\                  /\  \        /\  \
+                      /:/  /       ___        \:\  \       \:\  \
+                     /:/  /       /\__\        \:\  \       \:\  \
+                    /:/  /  ___  /:/__/    _____\:\  \  _____\:\  \
+                   /:/__/  /\__\/::\  \   /::::::::\__\/::::::::\__\
+                   \:\  \ /:/  /\/\:\  \__\:\~~\~~\/__/\:\~~\~~\/__/
+                    \:\  /:/  /    \:\/\__\\:\  \       \:\  \
+                     \:\/:/  /      \::/  / \:\  \       \:\  \
+                      \::/  /       /:/  /   \:\__\       \:\__\
+                       \/__/        \/__/     \/__/        \/__/
+
+```
+
+
+# CINN : Compiler Infrastructure for Neural Networks
+
+The project CINN is a machine learning compiler and executor for multiple hardware backends.
+It is designed to provide multiple layers of APIs to make tensor computation easier to define,  faster to execute, and more convenient to extend with hardware backends.
+Currently, it targets x86 CPUs and Nvidia GPUs.
+
+This project is under active development.
+
+## How it works
```

Comment:
Add a link to detail documents, the cinn folder in PaddlePaddle/docs

Reply:
done